src/Apps/IdentityVerification/IdentityVerificationPage.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, Button, Sans, Serif } from "@artsy/palette"
+import { IdentityVerificationPage_me } from "__generated__/IdentityVerificationPage_me.graphql"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { SystemContextConsumer, useTracking } from "Artsy"
+import * as Schema from "Artsy/Analytics/Schema"
+import { ErrorPage } from "Components/ErrorPage"
+import React from "react"
+import { Meta, Title as HeadTitle } from "react-head"
+import { createFragmentContainer } from "react-relay"
+import { graphql } from "relay-runtime"
+
+interface Props {
+  me: IdentityVerificationPage_me
+}
+
+export const IdentityVerification: React.FC<Props> = ({ me }) => {
```

Comment:
#minor - In our other apps the the main entry point file and export is named `SomeNameApp` (like `ArtistApp`, `ArtworkApp`, etc)